class.jetpack-onboarding-end-points.php

```diff
@@ -539,6 +545,41 @@ static function install_theme() {
 		wp_send_json_success( $theme_id );
 	}
 
+	static function install_plugin( $slug ) {
```

Comment:
We could borrow some of the newly introduced in WP `wp_ajax_install_plugin()`. For example, using `plugins_api()` allows us to omit the `WOOCOMMERCE_ID` class constant, allowing us to rely on just the plugin slug. Also, that function passes more detailed errors (for example if an error occurs within the filesystem).